--- feedback.module.orig	2009-06-15 13:38:09.000000000 +0200
+++ feedback.module	2009-06-16 11:38:27.000000000 +0200
@@ -142,6 +142,12 @@ function feedback_form() {
 
 function feedback_form_submit($form, &$form_state) {
   feedback_add_entry($form_state['values']['message'], $form_state['values']['location']);
+  
+  if (module_exists('rules')) {
+    global $user;
+    rules_invoke_event('feedback_add_entry', $user, $form_state['values']['message'], $form_state['values']['location']); 
+  }
+  
   $message = t('Thanks for your feedback!');
   if ($form_state['values']['ajax']) {
     echo drupal_to_js(array('message' => $message));
